org.sat4j.pb/src/main/java/org/sat4j/pb/SolverFactory.java

```diff
@@ -132,12 +132,12 @@ public static PBSolverCP newPBCPAllPB() {
         return newPBCP(new PBMaxDataStructure());
     }
 
-    /**
-     * @return Solver used to display in a string the pb-instance in OPB format.
-     */
-    public static IPBSolver newOPBStringSolver() {
-        return new OPBStringSolver();
-    }
+//    /**
```

Comment:
why remove this? doesnt seem like it would do any harm in being here?

Reply:
`OPBStringSolver` extends `DimacsStringSolver`, which doesn't seem to be present in this repository. I haven't looked into why yet though, so I took the simple route and just excluded every file with compile errors from the build.